lib/ecto/migration.ex

```diff
@@ -658,4 +659,17 @@ defmodule Ecto.Migration do
   defp validate_type!(%Reference{} = reference) do
     reference
   end
+
+  def prefix(opts_prefix) when is_nil(opts_prefix) do
```

Comment:
The macro can just call the function directly in runner: `Ecto.Migration.Runner.put_prefix(...)`. For this module itself, we can import it.